### DISCUSSION TITLE
How to move cursor to popup panel, I want to select the text on popup?

### DISCUSSION THREAD

**Discussion by ofzo** [helix-editor/helix#9455] [General] [answered]
<img width="1090" alt="image" src="https://github.com/helix-editor/helix/assets/18000037/1b4b3bce-cf2e-47de-a576-3705f38b2fd3">

**Comment by woojiq** [comment]
Afaik, no way. But you can simulate this, it's far from perfect:
1. Disable mouse support: `:set mouse false`.
2. Select + copy using a terminal.
3. Enable mouse: `:set mouse true`.

But be aware that it will not only highlight text in a popup, but entire lines of your terminal.

**Reply by ofzo** [reply]
Will this feature be considered for support in the future?

**Reply by woojiq** [reply, marked as answer]
Probably not. This documentation doesn't come out of nowhere, use `goto definitions` and copy from there.

Not exactly, but the same context:
https://github.com/helix-editor/helix/issues/2235#issuecomment-1107708468:
> It's a preview popup, I don't want to complicate things further by adding selection.

**Comment by nik-rev** [comment]
You can obtain the text using the PR: [feat: new command to open hover documentation in a new buffer instead of a popup by nik-rev · Pull Request #12208 · helix-editor/helix](https://github.com/helix-editor/helix/pull/12208)
